app/notify_client/__init__.py

```diff
@@ -64,14 +67,15 @@ def check_inactive_user(self, *args):
         # and we only want to check the first arg
         for arg in args:
             arg = str(arg)
-            if (
-                "get-login-gov-user" in arg
-                or "user/email" in arg
-                or "/activate" in arg
-                or "/email-code" in arg
-                or "/verify/code" in arg
-                or "/user" in arg
-            ):
+            if self.is_calling_signin_url(arg):
+                # if (
+                #     "get-login-gov-user" in arg
+                #     or "user/email" in arg
+                #     or "/activate" in arg
+                #     or "/email-code" in arg
+                #     or "/verify/code" in arg
+                #     or "/user" in arg
+                # ):
```

Comment:
Just double checking that we're sure we don't need to test all of these URL paths?  How was this confirmed?

And if that's the case, I think it's safe to remove the commented out code then to help clean things up!

Reply:
I did grep -ri "/email-code" ./**/**.py and verified that the uses of that partial path all start with "/user", then I checked the others.

I guess we could test invites and 90 day email verifications again

Reply:
Yes, we should make sure the invites and 90 day email verification work still with this.  We should be able to test in staging by modifying our user records in the DB to trigger the email verification again.  Thanks!